modules/vaos/spec/request/messages_request_spec.rb

```diff
@@ -4,45 +4,145 @@
 RSpec.describe 'vaos appointment request messages', type: :request do
   include SchemaMatchers
 
-  let(:request_id) { '8a4886886e4c8e22016e5bee49c30007' }
-
   before do
     Flipper.enable('va_online_scheduling')
     sign_in_as(current_user)
     allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
   end
 
-  context 'loa1 user with flipper enabled' do
-    let(:current_user) { build(:user, :loa1) }
+  describe 'GET messages' do
+    let(:request_id) { '8a4886886e4c8e22016e5bee49c30007' }
```

Comment:
seems like a mystery number

Reply:
it's the only request_id that would match what the VCR cassette has? The idea being that we record those and then have to play them back against the VCR cassette -- if they don't match the request would throw an error. Not sure how to address this. 🤔

Reply:
You can inject variable values into the VCR cassette at runtime.  [Here is a recent PR](https://github.com/department-of-veterans-affairs/vets-api/pull/3592/files#diff-cda0492bae902639014fa5d79a611bd2) with a very similar use case to yours where the author had to do this to get his specs/cassettes to work.